app.py

```diff
@@ -0,0 +1,656 @@
+from bson.objectid import ObjectId
+from datetime import datetime
+from flask_pymongo import PyMongo
+from dotenv import load_dotenv
+import os
+from flask import Flask, render_template, request, url_for, redirect, \
+    make_response, jsonify, flash, session
+
+# get environment variables
+load_dotenv()
+
+app = Flask(__name__)
+# app.config['MONGO_DBNAME'] = 'travelPal'
+app.config['MONGO_URI'] = os.getenv('MONGODB_URI')
+app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
+
+# initialise mongoDb
+mongo = PyMongo(app)
+
+# set collections variables
+users = mongo.db.users
+trips = mongo.db.trips
+stops = mongo.db.stops
+
+
+def checkUserPermission(checkLogin=True, checkTripOwner=False,
```

Comment:
The convention in python is to use snake case... `def check_user_permission` instead of `def checkUserPermission`

Reply:
Let's have this function in a separate file called utils.py (to represent utility functions) then import into this file... so our app.py only contains routes